components/library/lcms2/Makefile

```diff
@@ -46,12 +46,38 @@ CFLAGS             +=           $(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
 CXXFLAGS           +=           $(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
 LDFLAGS            +=           $(JPEG_LDFLAGS)
 
+# Seems that we have no other way to designate that we want large file support.
+CFLAGS += $(CPP_LARGEFILES)
```

Comment:
Have you checked that this didn't break ABI?

Reply:
I only tested eom, eog and xsane with the new lcms2. What should be done?

Reply:
Check, first of all, if previous 32-bit version of lcms2 was compiled with _FILE_OFFSET_BITS=64  (for example, in config.h).
This affects size of file-related structures (and their members) and can lead to interesting crashes. If it is not so, the safest thing to do is to build it with _FILE_OFFSET_BITS 64 and rebuild dependent apps.

Reply:
Given that poppler depends on lcms2 and IS compiled with _FILE_OFFSET_BITS=64, I'd added this option.

Reply:
I compared the headers and cannot find any trace of change after this flag is added, should be safe.